hack/lib/golang.sh

```diff
@@ -342,6 +342,7 @@ kubeedge::golang::small_build_place_binaries() {
   read -r ldflags <<< "$(kubeedge::version::ldflags)"
 
   mkdir -p ${KUBEEDGE_OUTPUT_BINPATH}
+  sudo chmod -R 777 ${KUBEEDGE_ROOT}/_output
```

Comment:
Looks like this is hard code for path, and do we need to set to `777`?

Reply:
yes, CI runs checks with username `runner`, and when building inside the container just with the below command
https://github.com/kubeedge/kubeedge/blob/697b8be4139ad86e08ad4c457210acfbc21e79f9/hack/make-rules/build_with_container.sh#L27-L29
the -v flag bind mount a volume, will assign the _output/local/bin directory to the root permission, and the user `runner` cannot access these directory.

Reply:
not know how to assign the _output/local/bin directory to the user `runner` permission, by test add flag `--user  $(id -u)` to this docker run command, it didn't work well.

Reply:
Not sure if it will work for adding `sudo` for `make smallbuild`?